templates/helm/orchestration-workflows/templates/clinvar.yaml

```diff
@@ -10,7 +35,7 @@ spec:
   resetValues: true
   chart:
     git: git://github.com/databiosphere/clinvar-ingest.git
-    ref: master
+    ref: ra-DSPDC-793-ingest-xml
```

Comment:
NOTE: Will revert back to master

Reply:
Are we running this branch as a one time thing?

Reply:
just showing what I had it pointed to to have it working correctly =) reverted back to master now